src/db/postgres/lrsql/postgres/sql/ddl.sql

```diff
@@ -281,6 +280,35 @@ CREATE TYPE scope_enum AS ENUM (
   'profile/read');
 ALTER TABLE IF EXISTS credential_to_scope ALTER COLUMN scope TYPE scope_enum USING (scope::scope_enum);
 
+/* Migration 2024-01-24 - Add profile and profile/read scopes */
+
+/* Simply adding the profile scope will cause a name clash with the reserved
+   OIDC profile scope. As a result, we add prefixes to create activity_profile
+   and agent_profile (and their read-only versions); this has the additional
+   benefit of narrowing the scope to just activity and agent profiles,
+   respectively. Since profile and profile/read have always been unused, we
+   are safe to remove them as enums. */
+
+-- :name alter-scope-enum-type-2!
+-- :command :execute
+-- :doc Add `activity_profile`, `activity_profile/read`, `agent_profile`, and `agent_profile/read` to `credential-to-scope.scope` enum.
+ALTER TABLE IF EXISTS credential_to_scope ALTER COLUMN scope TYPE VARCHAR(255);
+DROP TYPE IF EXISTS scope_enum;
```

Comment:
I assume this preserves existing references to the scopes? I'm pretty sure they just get stored as ints on the rows, but just wanted to be sure.

Reply:
I forgot the exact reason this was added in the first enum migration, but that's probably the reason why. I think if you don't include that line it would be similar to deleting table that is referenced by foreign keys - it'll get blocked.

Reply:
I just want to be certain that, whatever we do, we don't break existing data if an LRS updates.

Reply:
Given that these lines are present in the first migration, I have my doubts that stuff would break.